src/user/serializers.py

```diff
@@ -457,6 +459,12 @@ def get_author_profile(self, user):
             sentry.log_error(e)
             return {}
 
+    def get_rsc_earned(self, user):
+        return user.rsc_earned
```

Comment:
```suggestion
        rsc_earned = getattr(user, "rsc_earned", None)
        return rsc_earned
```
This ensures that it won't error out